doc/args.rst

```diff
@@ -1,13 +1,12 @@
-Command line
-------------
+Parametrizing execution
+=======================
 
-Main script of project `Universum` is ``__main__.py``.
-All command-line parameters, general and module-related, are passed to this entry point
-via ``{python} -m universum``.
+`Univesrum` can be executed via ``{python} -m universum`` with various parameters. These parameters can either be
```

Comment:
I am not sure you need to use "either .. or" construct here. It adds unnecessary contrast here.
Also, there is a minor inaccuracy: actually, not all parameters can be passed via environment variables.